Sources/AppcuesKit/Presentation/Debugger/FloatingView/DebugViewController.swift

```diff
@@ -57,8 +57,8 @@ internal class DebugViewController: UIViewController {
         debugView.setFloatingView(visible: true, animated: animated, programmatically: true)
     }
 
-    func hide(animated: Bool) {
-        debugView.setFloatingView(visible: false, animated: animated, programmatically: true)
+    func hide(animated: Bool, notify: Bool = false) {
+        debugView.setFloatingView(visible: false, animated: animated, programmatically: true, notify: notify)
```

Comment:
the `notify` is so we can hide the FAB while the confirmation model is up, and that hiding does not get interpreted as the user closing the debugger entirely, as a normal hide action would do.